### DISCUSSION TITLE
Examples for Rust?

### DISCUSSION THREAD

**Discussion by alicewith** [huggingface/safetensors#435] [Q&A]
I'm looking for the examples using Rust for safetensors, but I can't. Could anyone give me some useful ones? I think the documentation is not good so far.

**Comment by Narsil** [comment]
The docs in the README is strongly biased towards python for sure.

Have you tried looking at the rust doc: https://docs.rs/safetensors/0.4.2/safetensors/tensor/struct.SafeTensors.html#method.deserialize?

**Reply by alicewith** [reply]
So as far as I know, safetensors is just a tensor serialization format, which has no facilities to predict and inference any model, right?
